docs/data-sources/api_token.md

```diff
@@ -16,6 +16,11 @@ API Token data source
 data "astro_api_token" "example" {
   id = "clxm4836f00ql01me3nigmcr6"
 }
+
+# Output the API token value using terraform apply
+output "api_token" {
+  value = data.astro_api_token.example
```

Comment:
The token value shouldn't appear in any data sources
It's not returned in the API
Only on create and rotate

Reply:
Oh maybe the wording is weird
Maybe it is fine then
But when I first read it, i thought it was the actual value of the API token

Reply:
yeah i meant like all of the token metadata not the specific token value